rasa/__main__.py

```diff
@@ -99,8 +99,9 @@ def main() -> None:
     cmdline_arguments = arg_parser.parse_args()
 
     log_level = getattr(cmdline_arguments, "loglevel", None)
+    logging_config_file = getattr(cmdline_arguments, "logging-config-file", None)
```

Comment:
Shouldn't it read  the arg "logging_config_file" instead?

Reply:
The fix was released as of v`3.4.2` with the merge of this PR: https://github.com/RasaHQ/rasa/pull/11971

Reply:
oh, great, thank you!